.github/workflows/python_file_check.yml

```diff
@@ -0,0 +1,38 @@
+name: Python File Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  file_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Files Included Check
+        id: file_check
+        uses: andstor/file-existence-action@v2
+        with:
+          files: "README.md, CHANGELOG.md, pyproject.toml, tox.ini"
+
+      - name: Fail If Files Missing
+        if: steps.file_check.outputs.files_exists == 'false'
+        run: |
+          echo "You are missing one of the follwoing files"
+          echo "README.md, CHANGELOG.md, pyproject.toml, tox.ini"
+          echo "Please add the missing files and try again."
+          exit 1
+
+      - name: README Not Empty Check
```

Comment:
Should there be a similar check for CHANGELOG?

Reply:
Or https://github.com/marketplace/actions/check-changelog-for-changes

Reply:
No. Only because the CHANGELOG is going to be filled out by an action so just insuring it is present is enough. I suspect that the action would add one if it was missing but I can't be sure and adding an empty one isn't a problem. If you weren't using the auto release github action though, you'd have a point.

Reply:
https://github.com/google-github-actions/release-please-action